src/windows/map.c

```diff
@@ -936,8 +942,8 @@ static void window_map_inputsize_land(rct_window *w)
 
 static void window_map_inputsize_map(rct_window *w)
 {
-	((uint16*)TextInputDescriptionArgs)[0] = 50;
-	((uint16*)TextInputDescriptionArgs)[1] = 256;
+	((uint16*)TextInputDescriptionArgs)[0] = 13;
+	((uint16*)TextInputDescriptionArgs)[1] = 254;
```

Comment:
Ugh, I missed that, even though that's exactly what I created those defines for!